proto/api_key.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+
+import "proto/context.proto";
+
+package api_key;
+
+// An API key used to access BuildBuddy.
+message ApiKey {
+  // The unique ID of this API key.
+  // ex: "AK123456789"
+  string id = 1;
+
+  // The string value of this API key which is passed in API requests.
+  string value = 2;
+
+  // Optional. The user-specified label of this API key that helps them
+  // remember what it's for.
+  string label = 3;
+}
+
+message CreateApiKeyRequest {
+  context.RequestContext request_context = 1;
+
+  // The ID of the group to create the API key for.
+  // ex: "GR123456789"
+  string group_id = 2;
+}
+
+message CreateApiKeyResponse {
+  context.ResponseContext response_context = 1;
+
+  // The ID of the newly created key.
+  string id = 2;
```

Comment:
could we just return an ApiKey object here?

Reply:
Yeah that seems better - done!